codes/classical/Cycle Codes.yml

```diff
@@ -0,0 +1,34 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+
+# code id, physical, logical are all lower case
+code_id: cycle
+physical: bits
+logical: bits
+
+
+name: 'cycle code of graphs '
+introduced: '\cite{https://ieeexplore.ieee.org/document/394711}'
+
+
+description: 'Let \(G\) be a finite, undirected, connected graph without loops and multiple edges.The graph \(G\) has (\r\) vertices and \(n\) edges. Its parity check matrix (\H\) is defined as follows.(\H\) is an (\ r \times n\) matrix whose rows are indexed by the vertices of \(G\) and whose columns are indexed by the edges of \(G\), and such that \(H_{ij}=1\) if vertex \(i\) belongs to the edge \(j\) and \(H_{ij}=0\) otherwise.'
```

Comment:
Right now, this is hard to distinguish from a linear code on a graph. Mention relevance of "cycle" in the name of the code.

Reply:
Right: the cycle code is nothing but an F2 linear space spanned by the incidence vectors of the cycles in the set of edges. This may be a better definition.
The matrix in the above description spans a dual of the cycle code, called a cocycle or a cutset code of the graph.